src/consensus/consensus.h

```diff
@@ -22,4 +34,94 @@ enum {
     LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
 };
 
+/**
+ * Context-independent CTransaction validity checks.
+ * Nobody should spend an extra cycle on a transaction that doesn't pass this.
```

Comment:
@jtimon I am happy to review again, if you stick to what is mentioned in the commit message... I mean, it is great that you are adding documentation and fixing white space but it is a lot harder to verify MOVEONLY if there are random changes in between.

There is no rule that says you can only use one commit per pull request. Each goal which can be achieved in a separate commit, should indeed go into a separate atomic commit. (This is not meant specifically for you but also for others, including me.)